lib/sentry/processor/filter_request_body.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Sentry
+  module Processor
+    class FilterRequestBody < Raven::Processor
+      FILTERED_CONTROLLERS = %w[ppiu].freeze
+
+      def process(data)
```

Comment:
Our other processors (eg `pii_sanitizer`) have a public `process` method and a private `sanitize` method that implements the details. It makes sense to me to keep that consistent format.